console/bridge.go

```diff
@@ -172,22 +172,43 @@ func (b *bridge) UnlockAccount(call otto.FunctionCall) (response otto.Value) {
 	return val
 }
 
-// passwdByPrompt returns the password from a non-echoing password prompt.
-func (b *bridge) passwdByPrompt(call otto.FunctionCall, msg string) (passwd otto.Value) {
-	// If password is not given or is the null value, prompt the user for i
-	if call.Argument(1).IsUndefined() || call.Argument(1).IsNull() {
-		fmt.Fprintln(b.printer, msg)
-		if input, err := b.prompter.PromptPassword("Passphrase: "); err != nil {
-			throwJSException(err.Error())
-		} else {
-			passwd, _ = otto.ToValue(input)
+// passwdByPromptWithDuration returns the password and the duration from a non-echoing password prompt.
+func (b *bridge) passwdByPromptWithDuration(call otto.FunctionCall, msg string) (passwd, duration otto.Value) {
+	duration = otto.NullValue()
+	passwd = otto.NullValue()
+
+	lenArgs := len(call.ArgumentList)
+
+	if lenArgs > 2 {
```

Comment:
I think if statements in line 182~209 can be replaced with switch statement with following options. 
case 0:
case 1:
case 2:
default:

Reply:
reflected :)